lib/cdo/rack/cookie_dcdo.rb

```diff
@@ -0,0 +1,32 @@
+require 'json'
+require 'request_store'
```

Comment:
So bizarre to see this gem for personal reasons!! I think this gem came out of the rstat.us days for me... Steve wrote this when being plagued by switching between puma and thin. I can't remember which of us had to diagnose a thread-local-storage bug and running into this problem. A whole dang lifetime ago!!!!! Wow!!!!!

If we are using `RequestStore`, though, we should put it in the `Gemfile` proper. In the `development`/`test` group with `require: false`. I think it is only coincidentally there at the moment.

Reply:
https://github.com/code-dot-org/code-dot-org/pull/59142/commits/1b1b95380d98307f8abd8e2c37deea7736d55a60